### DISCUSSION TITLE
2022-03-09 Weekly Meeting 46 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#194] [Weekly Meetings]
Weekly Meeting 46 is happening on Wednesday March 9nd, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Announcements and Updates
-  We had a retro! We will have another in a few weeks. Here are some key takeaways: 
    - We are on day 30 of 90 toward project leadership discussion and need to spend the next 30 days working through the milestone: [Future Structure of OMI](https://github.com/omigroup/omigroup/milestone/2)
    - We need more maintainers (see future structure of OMI) 
    - "Its initially hard to tell which groups do what, and how I can get involved. I have to guess based on the title of some of the working groups. Like Metatraversal sounds like something I would like but I can’t tell what it is." Evo will update the documentation for groups missing readmes like metatraversal and interoperator. Groups should ensure your discord and github channels clearly state how to get involved. Let's make it easier to get more contributors! 
    - Ant has some bandwidth to help shake things up at OMI, give live demos, spread the word, etc. 
    - lukecarbis called out that our calendar was only subscribable in google calendar format, and [submitted a PR](https://github.com/omigroup/omigroup/issues/191#issuecomment-1061263395) to add other calendar formats. Supporters can now follow the OMI event calendar in other formats like ical. 

Working Group Updates:
-  OMI Website update
- OMI-Chan group
- Scripting group
- gLTF group
- any others?

Housekeeping:
Any?
- #176

Open Floor!
 
Please leave any additional agenda items below in the comments. Thanks!

**Comment by mrmetaverse** [comment]
## Last weeks retro summary

-  We had a retro! We will have another in a few weeks. Here are some key takeaways: 
    - We are on day 30 of 90 toward project leadership discussion and need to spend the next 30 days working through the milestone: [Future Structure of OMI](https://github.com/omigroup/omigroup/milestone/2)
    - We need more maintainers (see future structure of OMI) 
    - "Its initially hard to tell which groups do what, and how I can get involved. I have to guess based on the title of some of the working groups. Like Metatraversal sounds like something I would like but I can’t tell what it is." Evo will update the documentation for groups missing readmes like metatraversal and interoperator. Groups should ensure your discord and github channels clearly state how to get involved. Let's make it easier to get more contributors! 
    - Ant has some bandwidth to help shake things up at OMI, give live demos, spread the word, etc. 
    - lukecarbis called out that our calendar was only subscribable in google calendar format, and [submitted a PR](https://github.com/omigroup/omigroup/issues/191#issuecomment-1061263395) to add other calendar formats. Supporters can now follow the OMI event calendar in other formats like ical. 

We will have another retro in a few weeks and try an alternative time to accommodate more OMI contributors. If you would like to have a retro for your working group, and want help, please reach out to me @mrmetaverse

**Reply by mrmetaverse** [reply]
I suggest we explore and ultimately vote on minumum requirements for setting up a working group. #176 tracks this discussion.

**Comment by madjin** [comment]
The open metaverse movement is spread across diff servers, working on some meta-coordination ideas w/ avatar group
Avatar interop group isn't a formal OMI working group atm, but does aim to give feedback to OMI-scripting and gltf-extensions groups

Yesterday started an open metaverse community gcal, curating events that are spread across different communities
Dabbling with quickly adding new events via sesh.fyi from the M3 discord

![brave_L5BzM6RjcC](https://user-images.githubusercontent.com/32600939/157485711-f791a870-0305-4871-85bd-a23610b48053.png)

Eventual goal is to gear up with custom avatars gathered from diff corners of the metaverse and make an epic journey across different servers, spending quality time in each to connect with locals / establish outposts / share stories / exchange gifts / build bridges

![image](https://user-images.githubusercontent.com/32600939/157486553-24dfc42e-ed02-4e98-bfae-282e3b251ed4.png)

![image](https://user-images.githubusercontent.com/32600939/157486637-bc422aa5-beea-4fff-ad5e-014684ec9b92.png)

**Comment by mrmetaverse** [comment]
:bug: streaming on twitch and YouTube still says "interopability demo night" 

![Screenshot_20220309-120453.png](https://user-images.githubusercontent.com/63426722/157493194-b22a5a02-e553-4126-9864-57ceec34ce44.png)

**Comment by robertlong** [comment]
# Workshop Planning

## First workshop (March 16th, 2 hour meeting, 8:30 PST - 9:30 PST):

### Goals:
- Revamp leadership structure and community engagement
- Define Key Roles, Entities, and Relationships
  - Defining what these roles and entities are
  - Org Chart

## Week break for proposals / discussion  (March 23rd)

## Follow up workshop (March 30th)
- Organizational development
  - Applying results of first workshop to our org
  - Engagement strategy, making sure roles match the goals of the org

**Reply by mrmetaverse** [reply]
Please use the Milestone [Future structure of OMI](https://github.com/omigroup/omigroup/milestone/2) for this.

**Reply by mrmetaverse** [reply]
Added to milestone! 
https://github.com/omigroup/omigroup/issues/196
https://github.com/omigroup/omigroup/issues/197
